Test/VimCoreTest/InsertModeIntegrationTest.cs

```diff
@@ -2599,10 +2623,24 @@ public void WordCompletion_ChooseNext_Async()
                 _textView.MoveCaretTo(1);
                 _vimBuffer.ProcessNotation("<C-N>");
                 Dispatcher.DoEvents();
-                _vimBuffer.ProcessNotation("<C-N><CR>");
+                _vimBuffer.ProcessNotation("<C-N><C-y>");

```

Comment:
Nit: prefer `<C-Y>` here over `<C-y>` as that is the more common style in the code base.

Reply:
Huh. It seems the code and tests have a different style here. Strange. Would prefer the test at least use one of the styles vs. both. 

---
In reply to: [296050137](https://github.com/VsVim/VsVim/pull/2654#discussion_r296050137) [](ancestors = 296050137)

Reply:
All the mode maps use the lower case convention, and I thought I liked the lowercase convention, but the Vim docs systematically use the uppercase convention (and most of the unit tests). Our own `:map` command (to display mapped key sequences) uses the uppercase convention, as does Vim's `:map` command.

So if ought to be consistent, uppercase makes sense.